src/main/java/org/synyx/urlaubsverwaltung/absence/AbsenceServiceImpl.java

```diff
@@ -32,13 +40,39 @@
     private final ApplicationService applicationService;
     private final SickNoteService sickNoteService;
     private final SettingsService settingsService;
+    private final WorkingTimeService workingTimeService;
+    private final PublicHolidaysService publicHolidaysService;
 
     @Autowired
-    public AbsenceServiceImpl(ApplicationService applicationService, SickNoteService sickNoteService, SettingsService settingsService) {
+    public AbsenceServiceImpl(ApplicationService applicationService, SickNoteService sickNoteService,
+                              SettingsService settingsService, WorkingTimeService workingTimeService,
+                              PublicHolidaysService publicHolidaysService) {
 
         this.applicationService = applicationService;
         this.sickNoteService = sickNoteService;
         this.settingsService = settingsService;
+        this.workingTimeService = workingTimeService;
+        this.publicHolidaysService = publicHolidaysService;
+    }
+
+    @Override
+    public List<AbsencePeriod> getOpenAbsences(Person person, LocalDate start, LocalDate end) {
+        return getOpenAbsences(List.of(person), start, end);
+    }
+
+    @Override
+    public List<AbsencePeriod> getOpenAbsences(List<Person> persons, LocalDate start, LocalDate end) {
+        final DateRange askedDateRange = new DateRange(start, end);
+        final List<WorkingTime> workingTimeList = workingTimeService.getByPersonsAndDateInterval(persons, start, end);
+        final FederalState systemDefaultFederalState = workingTimeService.getSystemDefaultFederalState();
```

Comment:
wird hier noch irgendwo geschaut wenn der Benutzer eine anderes Bundesland spezifiziert hat?

Reply:
ich sehe in Zeile 269 wird das dann z.B. gemacht.